src/types/common/main.d.ts

```diff
@@ -20,3 +20,27 @@ export interface CustomizeBobaModalProps {
   modalShow: boolean;
   onHide: () => void;
 }
+
+export interface ToppingsProps {
+  name: string;
+  price: number;
+}
+
+export interface MilkProps {
+  name: string;
+  price: number;
+}
+
+export interface MilkList {
+  [key: string]: {
+    name: string;
+    price: number;
+  };
+}
+
+export interface ToppingList {
```

Comment:
`ToppingListProps`
If these exist and available, why not import into `src/assets/sampleBobaAPI.tsx` for use?

Reply:
i just posted there since sampleBobaAPI will go away eventually

Reply:
Okay please use ToppingListProps